### ISSUE TITLE
Crash with ProjectE + Baubles

### ISSUE THREAD

**Issue by juniorjunior** [sinkillerj/ProjectE#672]
ProjectE 1.4.4
Baubles 1.0.1.10
Forge 1.7.10_1381
Crash report: http://pastebin.com/Bb14KSR0

Placing SwiftWolf's Rending Gale into a Baubles ring slot crashes the client. This bug was introduced with ProjectE 1.4.4 as earlier versions (<=1.4.3.1) do not crash.

**Comment by MaPePeR** [comment]
Duplicate #669

**Comment by williewillus** [comment]
Duplicate of #655 and resolved by #659. Please search the tracker before reporting, but thanks for the report!

**Comment by juniorjunior** [comment]
Fair enough. Should have searched. Is there a Jenkins/build server for dev versions or do we just wait on a release?

**Comment by MaPePeR** [comment]
You wait for a release. there are no dev versions for your own safety, i think
